cost/turbonomics/buy_reserved_instances_recommendations/azure/turbonomics_buy_reserved_instances.pt

```diff
@@ -0,0 +1,304 @@
+name "Turbonomics Buy Reserved Instances Recommendations Azure"
+rs_pt_ver 20180301
+type "policy"
+short_description "Turbonomics policy for recommending purchaseable reserved instances [README](https://github.com/flexera-public/policy_templates/tree/master/cost/turbonomics/buy_reserved_instances_recommendations/azure) and [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
+severity "low"
+category "Cost"
+default_frequency "daily"
+info(
+  version: "2.0",
```

Comment:
Same comment as above (make this version number match with the one at the CHANGELOG.md)

Reply:
It is weird that in @midir99 example they start at 1.0: https://github.com/flexera-public/policy_templates/blob/master/operational/aws/long_running_instances/CHANGELOG.md

But in mine they used 2.0: https://github.com/flexera-public/policy_templates/blob/master/operational/aws/long_running_instances/CHANGELOG.md

I am not sure if there is a significant difference using one or another. @degarajesh, I think Jorge example is more related to your policy, so you might want to use his.

Reply:
Hi @Rangelmv23 -- The change to use `v0.1` for initial releases is recent and from the [discussion around switching to Semantic Versioning](https://teams.microsoft.com/l/message/19:833373548e104af2a20b0216eda1ba7b@thread.skype/1677611086855?tenantId=91034d23-0b63-4943-b138-367d4dfac252&groupId=fb250818-e040-4a26-b207-61c3cd99fd6e&parentMessageId=1677610206124&teamName=Team%20Flexera%20One&channelName=Policy%20Support%20and%20Questions&createdTime=1677611086855&allowXTenantAccess=false)

Reply:
@bryankaraffa addressed.